NthKey.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,9 @@
 
 /* Begin PBXBuildFile section */
 		1F6A2E586F1545C3C5EFA149 /* Pods_NthKey.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = CA4B65438CE45530547E491B /* Pods_NthKey.framework */; };
+		2AB30A0325EC00550061B717 /* UserDefaults.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2AB30A0225EC00550061B717 /* UserDefaults.swift */; };
```

Comment:
Why is this line added 3 times?

Reply:
@Sjors Because it added to all 3 targets: app/UnitTests/UITests. I think we should use the same code in tests too.